### PR TITLE
Adds column for quota utilization in client list page

### PR DESCRIPTION
**Pull request by bipuladh**
Jira Link: https://issues.redhat.com/browse/RHSTOR-6481

### CODE REVIEW

File: packages/odf/components/storage-consumers/client-list.tsx [hunk 1]
```diff
@@ -216,6 +225,9 @@ const ClientsList: React.FC<ClientListProps> = (props) => {
 const getOpenshiftVersion = (obj: StorageConsumerKind) =>
   obj?.status?.client?.platformVersion;
 
+const getStorageQuotaUtilizationRatio = (obj: StorageConsumerKind) =>
+  `${obj?.status?.client?.storageQuotaUtilizationRatio}`;
```
Comment (SanjalKatiyar) [inline packages/odf/components/storage-consumers/client-list.tsx:229]:
```suggestion
  obj?.status?.client?.storageQuotaUtilizationRatio;
```

**Comment by SanjalKatiyar** [comment]
@bipuladh can u plz add the Jira link to the PR description ?

File: packages/odf/components/storage-consumers/client-list.tsx (hunk 1 above)
Comment (bipuladh) [reply packages/odf/components/storage-consumers/client-list.tsx:229]: It needs to be string or JSX.Element number is neither

File: packages/odf/components/storage-consumers/client-list.tsx [hunk 2]
```diff
@@ -171,6 +175,11 @@ const ClientsList: React.FC<ClientListProps> = (props) => {
             column.sort = 'status.storageQuotaInGiB';
             column.props.info = { popover: <StorageQuotaPopoverContent /> };
             break;
+          case 'storageQuotaUtilRation':
```
Comment (alfonsomthd) [inline packages/odf/components/storage-consumers/client-list.tsx:178]: @bipuladh it should be `storageQuotaUtilRatio`

**Pull request merged**